scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -115,12 +120,93 @@ def strip_NaNs(output_workspace, base_output_name):
 
     return base_output_name
 
+##############################################################################
+# 26 March 2019
+
+
+def output_header(external_mode, used_wl_range, ws_sample, sample_thickness,
```

Comment:
This function is quite long and might benefit from being split up in multiple functions or separated by more comments, however it's in with the style of the file so I don't think there is a need to change.